### PR TITLE
VIDEO 10731 - Making addProcessor work across all supported browsers

### PR DESCRIPTION
**Pull request by charliesantos**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### Description

VIDEO 10731 - Making addProcessor work across all supported browsers
IDL https://code.hq.twilio.com/client/video-sdk-api/pull/102/files

### Before review
* [x] Updated CHANGELOG.md if necessary
* [ ] Added unit tests if necessary
* [x] Updated affected documentation
* [ ] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,41 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Version 1.x reached End of Life on September 8th, 2021.** See the changelog entry [here](https://www.twilio.com/changelog/end-of-life-complete-for-unsupported-versions-of-the-programmable-video-sdk). Support for the 1.x version ended on December 4th, 2020.
 
+2.23.1 (In Progress)
+====================
+
+Changes
+-------
+
+`VideoTrack.addProcessor` now works on browsers that doesn't support `OffscreenCanvas`. With this release, when used with [@twilio/video-processors
```
Comment (manjeshbhargav) [inline CHANGELOG.md:11]:
```suggestion
`VideoTrack.addProcessor` now works on browsers that do not support `OffscreenCanvas`. With this release, when used with [@twilio/video-processors
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,41 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
...
+
+const videoTrack = await createLocalVideoTrack({
+  width: 640,
+  height: 480,
+  frameRate: 24
+});
+
+videoTrack.addProcessor(processor, {
+  inputFrameBufferType: 'video',
+  outputFrameBufferContextType: 'webgl2',
```
Comment (manjeshbhargav) [inline CHANGELOG.md:36]: Let's use the enum type here, just to be consistent.

**Review by manjeshbhargav — APPROVED** [review]
@charliesantos LGTM.

**Comment by charliesantos** [comment]
Thanks @manjeshbhargav . I'll address the suggestions and write unit tests first before I merge.

**Pull request merged**